rest/data/host.go

```diff
@@ -73,7 +73,10 @@ func (hc *DBHostConnector) FindHostById(id string) (*host.Host, error) {
 func (hc *DBHostConnector) FindHostByIpAddress(ip string) (*host.Host, error) {
 	h, err := host.FindOne(host.ByIP(ip))
 	if err != nil {
-		return nil, err
+		h, err = host.FindOne(host.ByIPv4(ip))
```

Comment:
Should we maybe create a query that finds a host where either IPv4 or IPv6 equals ip?

Reply:
done